comid/measurement.go

```diff
@@ -320,15 +322,15 @@ func (o *Measurement) SetKeyPSARefValID(psaRefValID PSARefValID) *Measurement {
 	return o
 }
 
-// SetKeyCCARefValID sets the key of the target measurement-map to the supplied
+// SetKeyCCAPlatformConfigID sets the key of the target measurement-map to the supplied
 // CCA refval-id
-func (o *Measurement) SetKeyCCARefValID(ccaRefValID CCARefValID) *Measurement {
+func (o *Measurement) SetKeyCCAPlatformConfigID(ccaPlatformConfigID CCAPlatformConfigID) *Measurement {
 	if o != nil {
-		if ccaRefValID.Valid() != nil {
+		if ccaPlatformConfigID.Empty() {
```

Comment:
similar comment as per the unmarshaler / getter

Reply:
Here we do not return an error but a nil measurements!